samples/subsys/logging/syst/sample.yaml

```diff
@@ -28,14 +31,26 @@ tests:
     tags: logging
     harness: console
     harness_config:
-      type: one_line
+      type: multi_line
       regex:
         - "SYS-T RAW DATA: "
+        - "SYST Sample Execution Completed"
   sample.logger.syst.v2.immediate:
+    extra_args: OVERLAY_CONFIG=overlay_immediate.conf
     platform_allow: mps2_an385 qemu_x86 sam_e70_xplained
     tags: logging
     harness: console
     harness_config:
-      type: one_line
+      type: multi_line
       regex:
         - "SYS-T RAW DATA: "
+        - "SYST Sample Execution Completed"
+  sample.logger.syst.adsp_cavs:
+    extra_args: OVERLAY_CONFIG=overlay_cavs.conf
+    platform_allow: intel_adsp_cavs15 intel_adsp_cavs18 intel_adsp_cavs20 intel_adsp_cavs25
```

Comment:
doing this you are missing all the other tests on the listed platforms. You should add the overlay in boards/<board name> instead and remove all platform_allow entries here and instead using integration_platforms to allow more coverage.